### PR TITLE
Avoid executable file copying for regression tests

### PR DESCRIPTION
**Pull request by alex85k**
Following https://github.com/openstreetmap/osm2pgsql/pull/475

Added -x exe_name option to python test script (./osm2pgsql by default) and passed `` -x $<TARGET_FILE:osm2pgsql>`` in cmake test.

Checked on Windows.

### CODE REVIEW

File: tests/regression-test.py [hunk 1]
```diff
@@ -6,11 +6,14 @@
...
 import subprocess
 
+exe_path="./osm2pgsql"
 full_import_file="tests/liechtenstein-2013-08-03.osm.pbf"
 multipoly_import_file="tests/test_multipolygon.osm" #This file contains a number of different multi-polygon test cases
 diff_import_file="tests/000466354.osc.gz"
 diff_multipoly_import_file="tests/test_multipolygon_diff.osc" #This file contains a number of different multi-polygon diff processing test cases
 
+
+
```
Comment (pnorman) [inline tests/regression-test.py]: extra whitespace was added here

Comment (alex85k) [inline tests/regression-test.py]: Oops)

**Comment by alex85k** [comment]
Fixed the empty lines.

**Comment by alex85k** [comment]
Rebased & tested on Ubuntu 15.04 - OK.

**Pull request merged**